### PR TITLE
Uni 46254 mvp alembic ui

### PR DESCRIPTION
**Pull request by vkovec**
- hid timeline recorder + camera params

### CODE REVIEW

File: AlembicImporter/Packages/com.unity.formats.alembic/Editor/Timeline/AlembicRecorderClipEditor.cs [hunk 1]
```diff
@@ -156,3 +157,4 @@ TimelineAsset FindTimelineAsset()
         }
     }
 }
+#endif
```
Comment (lassond) [inline AlembicImporter/Packages/com.unity.formats.alembic/Editor/Timeline/AlembicRecorderClipEditor.cs]: Good practice is to add a comment following the endif, indicating what it relates to. Here for example:
#endif // ENABLE_ALEMBIC_TIMELINE_RECORDER

**Comment by lassond** [comment]
Does that cover the "end of frame" component? Is it gone from the component search dialog now?

**Comment by vkovec** [comment]
@lassond yes I think I added the #ifdef to that component as well

**Comment by vkovec** [comment]
@lassond unhiding the timeline recorder per Seiya's request

**Comment by vkovec** [comment]
ran the runtime tests and they all pass with these changes

**Comment by lassond** [comment]
Perfect, please merge!

**Pull request merged**